apps/deploy-web/tests/seeders/alert.ts

```diff
@@ -0,0 +1,70 @@
+import type { components } from "@akashnetwork/react-query-sdk/notifications";
```

Comment:
https://github.com/akash-network/console/blob/main/apps/notifications/test/seeders/chain-message-alert.seeder.ts
https://github.com/akash-network/console/blob/main/apps/notifications/test/seeders/deployment-balance-alert.seeder.ts

Reply:
I totally missed those, thanks. Used them and removed this seeder.

I was getting an ESLint error about the extraneous dependency:

```
error  '@akashnetwork/notifications' should be listed in the project's dependencies. Run 'npm i -S @akashnetwork/notifications' to add it  import-x/no-extraneous-dependencies
```
So I needed to add `@akashnetwork/notifications` package to deploy-web's workspace.